ee/modules/650-runtime-audit-engine/templates/k8s-metacollector/service.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: k8s-metacollector
+  namespace: d8-{{ $.Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "k8s-metacollector")) | nindent 2 }}
+spec:
+  ports:
+  - name: metrics
+    port: 8080
+    targetPort: metrics
+    protocol: "TCP"
+  - name: health-probe
```

Comment:
Why do we need to expose this port to the service?

Reply:
removed